### ISSUE TITLE
Activating "Message info" in context menu with keyboard does nothing

### ISSUE THREAD

**Issue by WofWca** [deltachat/deltachat-desktop#4242]
- Operating System (Linux/Mac/Windows/iOS/Android):
- Delta Chat Version: 9c5787410
- Expected behavior: the "message info" dialog gets opened
- Actual behavior: nothing apparently happens
- Steps to reproduce the problem:
    1. Right-click a message to open the context menu.
    2. Using the keyboard, navigate to "message info".
    3. Press Enter.
- Screenshots:
- Logs:

I found out that the dialog does get rendered, but it gets immediately closed, but _not_ by this code:

https://github.com/deltachat/deltachat-desktop/blob/0e0d0b80d598802ebb972c8d6c05b593bd896f24/packages/frontend/src/components/Dialog/Dialog.tsx#L32-L58

This issue does not appear in 1.46.8, I suspect this is a regression after #4006. This is not super important as of now since keyboard accessibility is lacking in other more important ways, but it will become important soon because we're working on accessibility now (#4210, #4224).

@maxphilippov would you like to take a look, or shall I?

**Comment by Simon-Laux** [comment]
I can reproduce this bug.

**Comment by WofWca** [comment]
There is another, probably related issue on 1.47.1.
1. Using keyboard, focus the "three-dot menu" button on a chat.
2. Press spacebar to open the menu
3. Select "Disappearing messages" with arrow keys.
4. Press enter to activate the menu item.

The dialog will open, but the menu will also remain open, and will retain focus.
